modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -84,18 +116,47 @@ spec:
         default: true
         objectStorage:
           bucket: <bucket_name>
-          prefix: <bucket_prefix>
+          prefix: <bucket-prefix>
         provider: aws
   configuration:
     restic:
       enable: <true_or_false>
     velero:
+       itemOperationSyncFrequency: "10s"
       defaultPlugins:
         - openshift
         - aws
         - csi
+        - vsm <1>
+  features:
+    dataMover:
+      credentialName: restic-secret
+      enable: true
+      maxConcurrentBackupVolumes: "3" <2>
+      maxConcurrentRestoreVolumes: "3" <3>
+      pruneInterval: "14" <4>
+      volumeOptions: <5>
+        sourceVolumeOptions:
+          accessMode: ReadOnlyMany
+          cacheAccessMode: ReadWriteOnce
+          cacheCapacity: 2Gi
+        destinationVolumeOptions:
+          storageClass: other-storageclass-name
+          cacheAccessMode: ReadWriteMany
+  snapshotLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-west-2
+        provider: aws
+
 ----
-<1> Add the Restic secret name from the previous step. If this is not done, the default secret name `dm-credential` is used.
+<1> OADP 1.2.x only.
+<2> OADP 1.2.x only. Optional: Specify the upper limit of the number of snapshots allowed to be queued for backup. The default value is 10.
+<3> OADP 1.2.x only. Optional: Specify the upper limit of the number of snapshots allowed to be queued for restore. The default value is 10.
+<4> OADP 1.2.x only. Optional: Specify the number of days, between running restic pruning on the repository. The prune operation repacks the data to free space, but it can also generate significant I/O traffic as a part of the process. Setting this option allows a trade-off between storage consumption, from no longer referenced data, and access costs.
```

Comment:
> restic pruning

Is this a known term? I see one reference on a quick internet search. Maybe _between pruning Restic backups_? At least, capitalize Restic, I think.

Reply:
Would it make sense to have two examples, 1.2.x and 1.1.x, to avoid copy/paste issues?

Reply:
1. pruneInterval is a new feature. I capitalized Restic, as you suggested.
2. I think if we had two examples it would make it more confusing for the user. Most users will only be using 1.2.x